cmd/replace.go

```diff
@@ -0,0 +1,171 @@
+// Copyright 2022 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os"
+	"regexp"
+
+	"github.com/spf13/cobra"
+
+	"github.com/hacbs-contract/ec-cli/internal/image"
+)
+
+type replaceFn func([]string, string) ([]byte, error)
+
+func replaceCmd(r replaceFn) *cobra.Command {
```

Comment:
Found the `r(...)` confusing to read

```suggestion
func replaceCmd(replace replaceFn) *cobra.Command {
```